client/src/pages/Chat.jsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import { useHistory } from 'react-router-dom';
+
+export default function Chat() {
+	const history = useHistory();
+
+	React.useEffect(
+		() => {
+			const userInfo = localStorage.getItem('currentUser');
```

Comment:
We can't rely just on the presence of the currentUser in localStorage to know that the user has logged in. This is because:
- the token may have expired 
- the user may have changed their information, or in the future, you have revoked their token on the back-end 

For the same reason, you should only store the token in localstorage and not the whole user. So instead, it's because to check the token, and send an API call to get the user to see if they are still logged to be certain that they logged in.

Reply:
I actually do have the user object available in my authentication server. Forgot to change this and will make these updates.